Samples/Samples.iOS/Info.plist

```diff
@@ -28,39 +28,21 @@
 	<key>CFBundleIdentifier</key>
 	<string>com.xamarin.essentials</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
-	<key>CFBundleIconFiles</key>
-	<array>
-		<string>Icon-60@2x</string>
-		<string>Icon-60@3x</string>
-		<string>Icon-76</string>
-		<string>Icon-76@2x</string>
-		<string>Default</string>
-		<string>Default@2x</string>
-		<string>Default-568h@2x</string>
-		<string>Default-Portrait</string>
-		<string>Default-Portrait@2x</string>
-		<string>Icon-Small-40</string>
-		<string>Icon-Small-40@2x</string>
-		<string>Icon-Small-40@3x</string>
-		<string>Icon-Small</string>
-		<string>Icon-Small@2x</string>
-		<string>Icon-Small@3x</string>
-	</array>
+	<string>0</string>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
 	<key>CFBundleName</key>
 	<string>Xamarin.Essentials</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>1.6.0</string>
```

Comment:
```suggestion
	<string>1.0</string>
```

Reply:
I'd prefer `1.6.0` here, but I can live with `1.0` as well.

One could also think about setting up a script that updates all version numbers in the project and config files for a release ...